README.md

```diff
@@ -40,15 +52,91 @@ or
 python setup.py develop
 ```
 
+## Getting started
+`s2spy` provides end-to-end solutions for machine learning (ML) based S2S forecasting.
+
+![workflow](./docs/assets/images/workflow.png)
+
+### Datetime operations & Data processing
+In a typical ML-based S2S project, the first step is always data processing.  A calendar-based datetime operation module is implemented to help the users prepare their data. For instance, if a user is looking for predictors for winter climate at seasonal timescales (~180 days), a `AdventCalendar` can be used to prepare the data:
+
+```py
+calendar = s2spy.time.AdventCalendar(anchor=(11, 30), freq='180d')
+calendar = calendar.map_years(2020, 2021)
+calendar
+>>>    i_interval                 (target) 0                         1
+>>>    anchor_year
+>>>    2021         (2021-06-03, 2021-11-30]  (2020-12-05, 2021-06-03]
+>>>    2020         (2020-06-03, 2020-11-30]  (2019-12-06, 2020-06-03]
+```
+
+`s2spy` works with `xarray` data. Now, the user can load their data and easily resample their data to the desired timescales configured in the calendar:
+
+```py
+calendar = calendar.map_to_data(input_data)
```

Comment:
I suggest renaming the variable `calendar` to something else here e.g. `mapped_calendar`.

Reply:
Thanks for the suggestion. But I think we just want to overwrite the old calendar with the mapped one and ask user to stick to it (Peter's idea). So in the tutorial notebook we also put it this way. I prefer to leave it like this. But thanks for your advice. I do see a point there 😉 .